website/docs/advanced-concepts/version-control-with-git/environments-with-git.md

```diff
@@ -1,41 +1,60 @@
-# Environments With Git
+# Multi-instance setup with Git 
 
-Appsmith enables you to create and maintain multiple versions of your applications and make changes to them in isolation using Git. Having separate staging and production workspaces ensures that users can continue to use your apps even while they're being changed or upgraded.
+This guide shows you how to set up multiple instances/environments in Appsmith using Git, making application development easier. Appsmith allows you manage multiple app versions, making development streamlined with separate staging and production workspaces.
 
 :::info
 This method doesn't support APIs, Google Sheets, Twilio, or any authenticated API datasources.
 :::
 
-Follow the steps below to setup the pipeline:
 
-### Setup environments
+## Prerequisites
 
-1. Create two workspaces for your staging and production versions of the app. These workspaces can be across two separate instances of Appsmith.
-2. Create your app in the staging workspace and [connect to a Git repository](/advanced-concepts/version-control-with-git/connecting-to-git-repository) . Once connected, check out a staging branch from the default branch(you can name the default branch as production).
+* Basic Knowledge of Git.
+* A self-hosted instance of Appsmith. Refer to the [Appsmith installation](/getting-started/setup/installation-guides) guides for detailed instructions on setting up your Appsmith instance.
 
-<dd>
+:::note
+If you're using two different self-hosted instances for production and staging, take a backup of both instances before updating Appsmith. Update only the staging instance and test all your apps on it before updating the production instance. In the event of an error, you can rollback by following the instructions on [Restore Appsmith Instance](/getting-started/setup/instance-management/appsmithctl#restore-instance)
+:::
 
-:::info 
-If you are an enterprise user, you can set a [Default branch](/advanced-concepts/version-control-with-git/working-with-branches#default-branch) and [Branch protection](/advanced-concepts/version-control-with-git/working-with-branches#branch-protection), ensuring code quality, collaboration, and stability in a collaborative development environment. 
+## Setup instances
 
-:::
+Follow these steps to set up multiple environments for your app:
 
-</dd>
+1. Create separate workspaces for each phase, such as staging and production.
+
+2. Create an app within the `staging` workspace and [connect to a Git repository](/advanced-concepts/version-control-with-git/connecting-to-git-repository). Once connected, check out a `staging` branch from the default branch.
+
+3. Navigate to the Appsmith homepage and [Import the app](/advanced-concepts/version-control-with-git/import-from-repository) into your `production` workspace using the **Import from Git** feature.
```

Comment:
The sentence structure sounds odd, please start with "In your production instance ......"